Contribute/how-to-write-docs-auth-pack.md

```diff
@@ -0,0 +1,89 @@
+---
+# required metadata
+
+title: Docs Authoring Pack for VS Code
+description: VS Code extension pack to facilitate Markdown authoring for docs.microsoft.com.
+author: meganbradley
+ms.author: mbradley
+manager: jemash
+ms.date: 04/06/2018
+ms.article: contributor-guide
+ms.prod: n.a
+ms.service: n.a
+ms.technology: n.a
+---
+# Docs Authoring Pack for VS Code
+
+The Docs Authoring Pack is a collection of VS Code extensions to aid with Markdown authoring for docs.microsoft.com. The pack is [available in the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) and contains the following extensions:
+
+- **DocFx:** Provides a docs.microsoft.com-specific Markdown preview. For more information, see [DocFx](https://marketplace.visualstudio.com/items?itemName=ms-docfx.DocFX).
+- **markdownlint:** A popular Markdown linter by David Anson to help ensure your Markdown follows best practices. For more information, see [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).
+- **Docs Markdown:** Provides Markdown authoring assistance for docs.microsoft.com content in the Open Publishing System (OPS), including basic Markdown support and support for custom Markdown syntax in OPS. The rest of this topic describes the Docs Markdown extension.
+
+## Prerequisites and assumptions
+
+To accurately insert relative links, images, and other embedded content with the Docs Markdown extension, you must have your VS Code workspace scoped to the root of your cloned OPS repo.
+
+Some syntax supported by the extension, such as alerts and snippets, are custom Markdown for OPS, and will not render correctly unless published via OPS.
+
+## How to use the Docs Markdown extension
+
+To access the Docs Markdown menu, type `ALT+M`. You can click or use up/down arrows to select the function you want, or type to start filtering, then hit `ENTER` when the function you want is highlighted in the menu. The following are available:
+
+|Function     |Command             |Description           |
+|-------------|--------------------|----------------------|
+|Bold         |`formatBold`        |Formats text **bold**.|
+|Italic       |`formatItalic`      |Formats text *italic*.|
+|Code         |`formatCode`        |If one line or less is selected, formats text as `inline code`.<br><br>If multiple lines are selected, formats them as a fenced code block, and allows you to optionally select a programming language supported by OPS.<br><br>If no language is selected, inserts an empty fenced code block.|
```

Comment:
3rd line, do you mean "If no lines are selected", instead of "if no language is selected"?